README.md

```diff
@@ -18,3 +18,21 @@ The Windows Azure transports for NServiceBus enable the use of Windows Azure Que
 - [Azure Storage Persistence](https://github.com/Particular/NServiceBus.Persistence.AzureStorage)
 - [Azure Cloud Services Host](http://docs.particular.net/nservicebus/hosting/cloudservices-host/)
 - [Samples](http://docs.particular.net/samples/azure/)
+
+## Third Party Dependency Update Policy
+
+The azure related nuget packages depend on third party packages. As each of these packages is maintained by different parties in different ways, our update policy regarding them also varies.
+
+The following aspects are taken into account:
+
+- Third party follows SemVer
+- API surface size
+- Past experience with behavioral changes in the dependency
+
+
+| Dependency                   | Policy                           |  Range          | 
+| ---------------------------- |---------------------------------:| ----------------|
+| WindowsAzure.ServiceBus      | Fixed Major                      |  [4.0.0, 5.0.0) |
+| WindowsAzure.Storage         | Fixed Major                      |  [8.0.0, 9.0.0) |
+| Newtonsoft.Json              | Fixed Major                      |  [10.0.0, 11.0.0)|
```

Comment:
With no agreement on Storage and Json libraries, should we move with this or not yet?